src/inmanta/data/model.py

```diff
@@ -873,13 +873,22 @@ def has_source(self) -> bool:
 
 
 class DataBaseReport(BaseModel):
+    """
+    :param max_pool: maximal pool size
+    :param free_pool: number of connections not in use in the pool
+    :param open_connections: number of connections currently open
+    :param free_connections: number of connections currently open and not in use
+    :param pool_exhaustion_time: nr of seconds since start we observed the pool to be exhausted
+    """
+
     connected: bool
     database: str
     host: str
     max_pool: int
+    free_pool: int
     open_connections: int
     free_connections: int
-    pool_exhaustion_count: int
+    pool_exhaustion_time: float
```

Comment:
This didn't really have a meaning, I changed it into `seconds of exhaustion` by taking into account the sample interval